pkg/controller/account/byoc.go

```diff
@@ -85,12 +88,21 @@ func (r *ReconcileAccount) initializeNewCCSAccount(reqLogger logr.Logger, accoun
 
 	accountID := account.Labels[fmt.Sprintf("%s", awsv1alpha1.IAMUserIDLabel)]
 
+	// Get SRE Access ARN from configmap
+	cm := &corev1.ConfigMap{}
+	cmErr := r.Client.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: awsv1alpha1.DefaultConfigMap}, cm)
+	if cmErr != nil {
+		reqLogger.Error(cmErr, "There was an error getting the ConfigMap to get the SRE Access Role")
+		return "", reconcile.Result{}, cmErr
+	}
+	SREAccessARN := cm.Data["CCS-Access-Arn"]
```

Comment:
Thoughts on adding validation here? Maybe someone changed or removed from the CM after opreator start up

Reply:
Good idea, I'll do that.

Reply:
Updated.  If you want to test this specifically, 

* `operator-sdk up local --namespace aws-account-operator
* in another terminal: `oc edit configmap -n aws-account-operator aws-account-operator-configmap` and remove the CCS-Access-Arn line.
* `make test-ccs` should now fail with the new error.